engine/src/state_chain_observer/client/extrinsic_api/signed/submission_watcher.rs

```diff
@@ -566,6 +566,7 @@ impl<'a, 'env, BaseRpcClient: base_rpc_api::BaseRpcApi + Send + Sync + 'static>
 								matching_request.until_in_block_sender
 							{
 								let _result = until_in_block_sender.send(
+									#[allow(clippy::useless_asref)]
 									result.as_ref().map(Clone::clone).map_err(
```

Comment:
I assume you tried removing the as_ref()?

Reply:
Yep.

Reply:
I guess I can add a comment.

Reply:
no need